wireguard/wireguard_freebsd.go

```diff
@@ -124,47 +68,114 @@ func create(nc *NCIface) error {
 			}
 		}
 	}
-	//set MTU
-	if _, err := ncutils.RunCmd(ifconfig+" "+nc.Name+" mtu "+strconv.Itoa(nc.MTU), true); err != nil {
-		return fmt.Errorf("error setting mtu %w", err)
-	}
-	if _, err := ncutils.RunCmd(ifconfig+" "+nc.Name+" up", true); err != nil {
-		return fmt.Errorf("error bringing up interface %w", err)
-	}
 	return nil
 }
 
-func strip() error {
-	in, err := os.Open(config.GetNetclientPath() + "netmaker.conf")
-	if err != nil {
-		return err
-	}
-	defer in.Close()
-	out, err := os.Create(os.TempDir() + "/netmaker.conf")
+// netlink.SetMTU - set MTU for netmaker interface
+func (nc *NCIface) SetMTU() error {
+	ifconfig, err := exec.LookPath("ifconfig")
 	if err != nil {
+		logger.Log(0, "failed to locate ifconfig", err.Error())
 		return err
 	}
-	defer out.Close()
-	writer := bufio.NewWriter(out)
-	scan := bufio.NewScanner(in)
-	scan.Split(bufio.ScanLines)
-	for scan.Scan() {
-		line := scan.Text()
-		if strings.Contains(strings.ToLower(line), "address") {
-			continue
-		}
-		if strings.Contains(strings.ToLower(line), "mtu") {
-			continue
-		}
-		if strings.Contains(strings.ToLower(line), "dns") {
-			continue
-		}
-		fmt.Fprint(writer, line+"\n")
+	//set MTU
+	if _, err := ncutils.RunCmd(ifconfig+" "+nc.Name+" mtu "+strconv.Itoa(nc.MTU), true); err != nil {
+		return fmt.Errorf("error setting mtu %w", err)
 	}
-	writer.Flush()
 	return nil
 }
 
-func apply(n *config.Node, c *wgtypes.Config) error {
-	return nil
-}
+// Configure does nothing on freebsd, all configuration is done by NCIface.Create()
```

Comment:
is there a purpose to keeping the commented out functions?

Reply:
removed